addons/purchase/data/mail_template_data.xml

```diff
@@ -85,6 +85,8 @@
         is expected for 
         % if object.date_planned:
             <strong>${format_date(object.date_planned)}</strong>.
+        % else:
```

Comment:
I don't really like this fix. I asked Oriane if we could rather  change the sentence in case of date_planned not set.

Reply:
Since this will only happen when you send a preview to yourself not to the vendor, Oma decided to keep this way first.